pyproject.toml

```diff
@@ -63,3 +63,86 @@ editor = [
 homepage = "https://github.com/goodmami/wn"
 documentation = "https://wn.readthedocs.io"
 changelog = "https://github.com/goodmami/wn/blob/main/CHANGELOG.md"
+
+[tool.ruff]
+select = [
+  "AIR",    # Airflow
+  "ASYNC",  # flake8-async
+  "B",      # flake8-bugbear
+  "BLE",    # flake8-blind-except
+  "C4",     # flake8-comprehensions
+  "C90",    # McCabe cyclomatic complexity
+  "CPY",    # flake8-copyright
+  "DJ",     # flake8-django
+  "DTZ",    # flake8-datetimez
+  "E",      # pycodestyle
+  "EXE",    # flake8-executable
+  "F",      # Pyflakes
+  "FLY",    # flynt
+  "G",      # flake8-logging-format
+  "ICN",    # flake8-import-conventions
+  "INT",    # flake8-gettext
+  "ISC",    # flake8-implicit-str-concat
+  "NPY",    # NumPy-specific rules
+  "PD",     # pandas-vet
+  "PERF",   # Perflint
+  "PGH",    # pygrep-hooks
+  "PIE",    # flake8-pie
+  "PL",     # Pylint
+  "PT",     # flake8-pytest-style
+  "PTH",    # flake8-use-pathlib
+  "PYI",    # flake8-pyi
+  "RSE",    # flake8-raise
+  "RUF",    # Ruff-specific rules
+  "S",      # flake8-bandit
+  "SIM",    # flake8-simplify
+  "SLOT",   # flake8-slots
+  "T10",    # flake8-debugger
+  "TCH",    # flake8-type-checking
+  "TID",    # flake8-tidy-imports
+  "UP",     # pyupgrade
+  "W",      # pycodestyle
+  "YTT",    # flake8-2020
+  # "A",    # flake8-builtins
+  # "ANN",  # flake8-annotations
+  # "ARG",  # flake8-unused-arguments
+  # "COM",  # flake8-commas
+  # "D",    # pydocstyle
+  # "EM",   # flake8-errmsg
+  # "ERA",  # eradicate
+  # "FA",   # flake8-future-annotations
+  # "FBT",  # flake8-boolean-trap
+  # "FIX",  # flake8-fixme
+  # "I",    # isort
+  # "INP",  # flake8-no-pep420
+  # "N",    # pep8-naming
+  # "Q",    # flake8-quotes
+  # "RET",  # flake8-return
+  # "SLF",  # flake8-self
+  # "T20",  # flake8-print
+  # "TD",   # flake8-todos
+  # "TRY",  # tryceratops
+]
```

Comment:
Better safe than sorry.  Ruff can lint the entire CPython codebase in 1/4 second.  Just because this codebase does not currently use Django or Pandas does not mean that they will never used.  If the tests pass in the blink of an eye then why not run them just to be sure?

An alternative would be decrease the list with `--select=ALL --ignore=A,ANN,ARG...` but that would allow new classes of tests to pop in unannounced.  `FURB` was recently added.

Reply:
> Better safe than sorry

The principle I'd rather apply here is [YAGNI](https://en.m.wikipedia.org/wiki/You_aren%27t_gonna_need_it)

> Just because this codebase does not currently use Django or Pandas does not mean that they will never used.

Django will never be used. It's not that kind of project.

Pandas may be a bit more likely, but in that case, I'd rather add linting for it when it's added than anticipating a possible future where it may be needed.

> If the tests pass in the blink of an eye then why not run them just to be sure?

To be sure of what? Ruff's speed is impressive, sure, but that doesn't mean I want to lint for things that don't exist just because I won't be inconvenienced by the delay.